src/TaxToken.sol

```diff
@@ -153,6 +158,8 @@ contract TaxToken {
     /// @dev Emitted when transferOwnership() is completed.
     event OwnershipTransferred(address indexed _currentAdmin, address indexed _newAdmin);
 
+    /// @dev Emitted when updating authorized users.
+    event UpdatedAuthorizedUsers(address indexed _account, bool _state);

```

Comment:
Maybe change "users to "wallets" considering this list might contain a smart contract address, not just users.